src/home_robot/home_robot/core/interfaces.py

```diff
@@ -3,6 +3,7 @@
 from typing import Any, Dict, Optional
 
 import numpy as np
+from habitat.core.simulator import Observations
```

Comment:
I thought we wanted to avoid dependency on habitat here?

if it's strictly necessary we can add it

Reply:
@exhaustin can you comment on how we should design this to avoid dependencies

Reply:
Yes, I plan to remove this particular one

Reply:
This is removed now